assets/media/authors/assanedi.markdown

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Move this file into _community_members directory. See an example community member profile https://github.com/opensearch-project/project-website/blob/main/_community_members/_sample.md.